### ISSUE TITLE
stats: `atuin stats year` raises an error

### ISSUE THREAD

**Issue by philn** [atuinsh/atuin#1475]
```
Error: The system's UTC offset could not be determined

Location:                                                                                                                                                                                                           
    /var/home/phil/.cargo-home/registry/src/index.crates.io-6f17d22bba15001f/atuin-17.1.0/src/command/client/stats.rs:102:23
```

```
timedatectl show   
Timezone=Atlantic/Canary
LocalRTC=no
CanNTP=yes
NTP=yes
NTPSynchronized=no
TimeUSec=Mon 2024-01-01 14:55:58 UTC
RTCTimeUSec=Mon 2024-01-01 14:55:59 UTC
```

**Comment by philn** [comment]
`month` and `week` sub-commands raise the same error.

**Comment by ellie** [comment]
resolved in #1477

**Comment by 0x4A6F** [comment]
Thanks for this report.
And also thanks for that fast fix by @conradludgate 

Happy New Year and let the `atuin stats year` reports flow freely.
